backend/app/services/synthesis/utils.py

```diff
@@ -0,0 +1,173 @@
+"""
+Utility functions and constants for the synthesis workflow.
+
+Contains normalisation helpers, string escaping, document info extraction,
+and Langfuse configuration utilities.
+"""
+
+from __future__ import annotations
+
+import re
+from typing import Dict, List, Optional, Any, TYPE_CHECKING
+
+from app.utils.llm.llm_utils import build_langfuse_metadata
+
+if TYPE_CHECKING:
+    from app.services.synthesis.state import SynthesisState
+
+
+# Model selection constants
+THEME_MODEL = "gpt-5-mini"
+MAPPING_MODEL = "gpt-5-nano"
+BRIEFING_MODEL = "gpt-5"
+
+# Study type code mapping (Maryland Scientific Methods Scale - matches extraction prompts)
+STUDY_TYPE_LABELS: Dict[str, str] = {
+    "a": "Cross-Sectional",
+    "b": "Pre-Post Study",
+    "c": "Cross-Sectional with Controls",
+    "d": "Pre-Post with Controls",
+    "e": "Comparison Study",
+    "f": "Quasi-Experimental",
+    "g": "RCT",
+    "h": "Meta-Analysis",
+    "i": "Policy/Theoretical",
+    "j": "Non-Scientific",
+}
```

Comment:
Just a note to myself and @helloaidank that we'll likely want to switch these to the new categorisation once that's ready.